src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -1136,57 +1136,82 @@ class UiaTextRangeTests
         // the UTR should refuse to move past the end.
 
         const auto bufferSize{ _pTextBuffer->GetSize() };
-        const til::point endInclusive = { bufferSize.RightInclusive(), bufferSize.BottomInclusive() };
-        const auto endExclusive{ bufferSize.EndExclusive() };
+        const COORD origin{ bufferSize.Origin() };
+        const COORD lastLineStart{ bufferSize.Left(), bufferSize.BottomInclusive() };
+        const COORD secondToLastCharacterPos{ bufferSize.RightInclusive() - 1, bufferSize.BottomInclusive() };
+        const COORD endInclusive{ bufferSize.RightInclusive(), bufferSize.BottomInclusive() };
+        const COORD endExclusive{ bufferSize.EndExclusive() };
 
-        // Iterate over each TextUnit. If the we don't support
+        // Iterate over each TextUnit. If we don't support
         // the given TextUnit, we're supposed to fallback
         // to the last one that was defined anyways.
+        BEGIN_TEST_METHOD_PROPERTIES()
+            TEST_METHOD_PROPERTY(L"Data:textUnit", L"{0, 1, 2, 3, 4, 5, 6}")
+            TEST_METHOD_PROPERTY(L"Data:degenerate", L"{false, true}")
+        END_TEST_METHOD_PROPERTIES();

```

Comment:
great catch!

Reply:
Thanks! Makes the tests so much more legible too haha.